### PR TITLE
Core api delete account

### PR DESCRIPTION
**Pull request by ChrisOngCC**
Per Iuvany, a script was provided to me and asked to be uploaded to GitHub, this is not my script.  It was given to me in a Zip file with other data files, all non-python files have been omitted, script has not been altered by me either, although, I think it can use an update with its comments as it seems dated and/or looks to have been altered at some point from user to account mangement.

### CODE REVIEW

**Comment by ChrisOngCC** [comment]
Please see my comment above, not sure who (customer) asked for this on GH, but if you think further review should be done on the functionality of the script/documentation, I can review/test, else, per Iuvany's request, I will merge PR.

**Comment by ChrisOngCC** [comment]
Added an example log file and data file with instructions on how to use the script and generate the data file

File: Offboarding/delete_accounts.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    Note:  The first row is for column names
+3) To call this script, use the following: python3 delete_accounts.py <admin level access key>
+4) Please ensure that this file is in the same folder/directory as the data file.
+"""
+
+def log_information(log_string):
+    logging.info(log_string)
+
+
+def write_users_created(users_output, users_recorder):
```
Comment (alecrajeev) [inline Offboarding/delete_accounts.py:29]: Don't think this function is required for this script

File: Offboarding/delete_accounts.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    return accounts_list
+
+def delete_account(env, admin_api_key, account_name):
+
+    api_url = env + "/api/account.json/delete_account"
+
+    delete_user_info = json.dumps({"account_name": account_name})
+
+    r7 = requests.post(api_url, headers = {"Content-Type": "application/json", "access_key": admin_api_key}, data = delete_user_info)
```
Comment (alecrajeev) [inline Offboarding/delete_accounts.py:45]: Would be good to wrap this in a try/catch block. If one API call fails, it should log the error and continue on.

File: Offboarding/logging_for_deleting_accounts.log [hunk 3]
```diff
@@ -0,0 +1,11 @@
+INFO:root:                   Environment name of the account in CloudCheckr
+0  https://api.cloudcheckr.com                    test46534534545
+1  https://api.cloudcheckr.com                        testaccount
+INFO:root:Started deleting accounts
+INFO:root:https://api.cloudcheckr.com
```
Comment (alecrajeev) [inline Offboarding/logging_for_deleting_accounts.log:5]: Don't think it needs to log the environment multiple times

File: Offboarding/example_datafile_delete_accounts.csv [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Environment,name of the account in CloudCheckr
+https://api.cloudcheckr.com,Example Payer
```
Comment (alecrajeev) [inline Offboarding/example_datafile_delete_accounts.csv]: I like that the example file was added!

**Comment by alecrajeev** [comment]
I think a nice readme would be helpful as well. Happy to hop on a quick zoom to review any questions.

File: Offboarding/logging_for_deleting_accounts.log (hunk 3 above)
Comment (ChrisOngCC) [reply Offboarding/logging_for_deleting_accounts.log:5]: I think I removed the whole logging thing for this part, just going to log the API request parameters so each call is logged, whether pass or fail

File: Offboarding/delete_accounts.py (hunk 2 above)
Comment (ChrisOngCC) [reply Offboarding/delete_accounts.py:45]: Success will log 200 + the endpoint and request parameters to identify which data row corresponds with the request.  

Failure will log the endpoint + status code, the request parameters, and hopefully return an ident message for support to debug.

**Comment by ChrisOngCC** [comment]
@alecrajeev changes completed, pending your review

**Pull request merged**